src/NuGet.Services.Metadata.Catalog.Monitoring/Status/PackageMonitoringStatusService.cs

```diff
@@ -28,40 +26,69 @@ public class PackageMonitoringStatusService : IPackageMonitoringStatusService
         /// </summary>
         private IStorageFactory _storageFactory;
 
-        /// <summary>
-        /// The <see cref="JsonSerializerSettings"/> to use to save and load statuses of packages.
-        /// </summary>
-        private static JsonSerializerSettings SerializerSettings => _serializerSettings.Value;
-        private static Lazy<JsonSerializerSettings> _serializerSettings = new Lazy<JsonSerializerSettings>(() =>
-        {
-            var settings = new JsonSerializerSettings();
-
-            settings.Converters.Add(new NuGetVersionConverter());
-            settings.Converters.Add(new StringEnumConverter());
-
-            return settings;
-        });
-
         public PackageMonitoringStatusService(IStorageFactory storageFactory, ILogger<PackageMonitoringStatusService> logger)
         {
             _logger = logger;
             _storageFactory = storageFactory;
         }
 
-        public async Task<PackageMonitoringStatus> GetAsync(FeedPackageIdentity package, CancellationToken token)
+        public async Task<IEnumerable<PackageMonitoringStatusListItem>> ListAsync(CancellationToken token)
         {
```

Comment:
The token seems that  it is not used .

Reply:
it's used by `var list = await storage.List(token);`

i will add it to the `Task.Run` call as well.